pyphare/pyphare/pharein/simulation.py

```
@@ -534,6 +534,23 @@ def check_restart_options(**kwargs):
     return restart_options
 
 
+def check_init_options(**kwargs):
+    """Advanced options to initialize from SAMRAI HDF5 files"""
+
+    formats = ["samraih5"]
+    init_options = kwargs.get("init_options", None)
```

Ruff

Use `kwargs.get("init_options")` instead of `kwargs.get("init_options", None)`

Replace `kwargs.get("init_options", None)` with `kwargs.get("init_options")`

(SIM910)